Remove unreachable code2

jrafanie opened the pull request:
SharedOps.renameDisks returns at the beginning so it's not needed. …
rubocop --only Lint/UnreachableCode

This may conflict with #1289, but it appears to not be included in that PR.  cc @gmcculloug

kbrock commented:
@gmcculloug Any pointers how much of Verbs is still needed?

gmcculloug commented:
@kbrock Most of Verbs can be removed with the exception of a few items in MiqVerbs.rb and SharedOps.rb required for fleecing.  Methods like SyncMetadata, ScanMetadata, UpdateAgentState and AgentJobState are still used.

Fryguy commented:
@jrafanie What is the history on why this was NOOPed?

jrafanie commented:
I believe this was the first comment/NO-OP commit:
```
commit d35cd8c2212b81f423a78da67dd423dcd7be0b89
Date:   Tue May 22 15:53:26 2007 +0000

    Changed to create blackbox on vm register and store vmId in it.
    Removed the rename of vm disk files when making a vm "smart".
    BugzID: 346
```

Fryguy commented:
wow 2007 and BugzID 364 :dizzy_face:

jrafanie commented:
@Fryguy @gmcculloug @kbrock Is anything else needed for this PR?

gmcculloug commented:
Nope, I think we are good here.

jrafanie commented:
Yay!  :tada: Thanks!
